driver/razeraccessory_driver.c

```diff
@@ -1020,6 +1098,12 @@ static ssize_t razer_attr_read_device_serial(struct device *dev, struct device_a
         strncpy(&serial_string[0], &response.arguments[0], 22);
         serial_string[22] = '\0';
         break;
+    case USB_DEVICE_ID_RAZER_MOUSE_DOCK_PRO:
+        request.transaction_id.id = 0xFF;
+        razer_send_payload(device, &request, &response);
+        strncpy(&serial_string[0], &response.arguments[0], 22);
+        serial_string[22] = '\0';
+        break;
```

Comment:
This is the same as the case with `USB_DEVICE_ID_RAZER_MOUSE_DOCK` and the others